android/src/main/java/com/reactnativecommunity/picker/ReactPickerManager.java

```diff
@@ -149,9 +149,15 @@ protected void onAfterUpdateTransaction(ReactPicker view) {
   protected void addEventEmitters(
       final ThemedReactContext reactContext,
       final ReactPicker picker) {
+    EventDispatcher eventDispatcher =
```

Comment:
Why is it done like this now? Cause it is a null on `Fabric`? If so, what about those listeners?